config/routes.rb

```diff
@@ -11,8 +11,10 @@
     collection do
       get '/find_reservations_by_user', to: "reservations#find_reservations_by_user"
       get '/find_reservations_by_seance', to: "reservations#find_reservations_by_seance"
+      patch '/cancel_reservation', to: "reservations#cancel_reservation"
+      patch '/confirm_reservation', to: "reservations#confirm_reservation"
```

Comment:
You already have in the path the word "reservation" (check `rails routes`!). WDYT about staying with just "cancel" and "confirm"?
```suggestion
      patch '/cancel', to: "reservations#cancel_reservation"
      patch '/confirm', to: "reservations#confirm_reservation"
```

Reply:
BTW Why didn't you define these methods as members of the resource? Please notice that you would like to invoke some operations on **a particular resource**, **not on the whole collection**.

Reply:
I renamed the paths, since it really was unnecessary to have reservation in them.
Also defined the methods as a member. The collection here really did not make sense.